openshift/unicorn-base/docker/Dockerfile

```diff
@@ -1,15 +1,14 @@
-FROM python:3.8-buster
+# As per https://fastapi.tiangolo.com/deployment/, we use the provided docker image:
+FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8
+# see: https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker for available images.
 
-# spotwx has an old certificate, so we have to make debian more forgiving.
-RUN sed -i 's/TLSv1.2/TLSv1.0/g' /etc/ssl/openssl.cnf
-
-# Install dependancies need by python developer packages
-# NOTE: Once we no longer need pyodbc, please remove the apt-get update and install commands below.
-RUN apt-get -y update
-RUN apt-get -y install unixodbc-dev
 # Install old (2.4.*; current debian) version of gdal
+RUN apt-get -y update
 RUN apt-get -y install libgdal-dev
 
+# spotwx has an old certificate, so we have to make debian more forgiving.
```

Comment:
is this needed anymore, now that we're not using spotwx?

Reply:
Thanks I missed updating the text. Wildfire was actually being hosted on an endpoint that also needed this fix - but I've gone and commented it out for now. When we get around to integrating with them again, we can put it back in we we need to.